e2e/browser/src/components/appContainer/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useEffect } from "react";
+import {
+  login,
+  logout,
+  handleIncomingRedirect,
+  getDefaultSession,
+} from "@inrupt/solid-client-authn-browser";
+import Notifications from "../notifications/";
+
+const REDIRECT_URL = "http://localhost:3000/";
```

Comment:
probably better to just use `window.location.href` here

Reply:
c3d761f15ee49e1d0154324348f60167883d2d65